### PR TITLE
Document Zeebe APIs for dual-region operations

### PR DESCRIPTION
**Pull request by deepthidevaki**
## Description

Related to https://github.com/camunda/camunda/issues/16126 

This PR documents 
1. The new features added to cluster scaling API for supporting the dual-region deployment
2. The new API for enabling and disabling the exporter

How to use these APIs are already documented in https://github.com/camunda/camunda-docs/pull/4205 

## When should this change go live?

With the next 8.6 alpha or 8.6 minor which ever is the earliest.

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by mesellings** [comment]
@deepthidevaki Which alpha/release is this for please?

**Comment by deepthidevaki** [comment]
> @deepthidevaki Which alpha/release is this for please?

If possible, include in the next 8.6 alpha (next week). Otherwise, in 8.6 minor.

File: docs/self-managed/zeebe-deployment/operations/cluster-scaling.md [hunk 1]
```diff
@@ -369,6 +369,28 @@ curl --request POST 'http://localhost:9600/actuator/cluster/brokers?dryRun=true'
...
+
+```
+curl --request POST 'http://localhost:9600/actuator/cluster/brokers?replicationFactor=4' \
+-H 'Content-Type: application/json' \
+-d '["0", "1", "2", "3"]'
+```
+
+#### Force remove brokers
+
+When some brokers are unreachable, you may want to remove them from the cluster. Usually, you can make changes to the cluster only when all brokers are up. However you can force remove a set of brokers by setting the request parameter `force` to `true`. This is mainly useful for the dual-region setup. For more details, read the [operational procedure for dual-region](/self-managed/operational-guides/multi-region/dual-region-ops.md/).
```
Comment (lenaschoenburg) [inline docs/self-managed/zeebe-deployment/operations/cluster-scaling.md:384]: :wrench: Make it clearer that this is a dangerous thing and it's on the user to ensure things such as force-removed brokers must not suddenly reconnect?

File: docs/self-managed/zeebe-deployment/operations/management-api.md [hunk 2]
```diff
@@ -61,3 +62,61 @@ When all partitions soft pause exporting, a successful response is received. If
...
+
+<TabItem value="disable">
+
+To disable an exporter, send the following request to the gateway's management API.
+
+```
+POST actuator/exporters/{exporterId}/disable
+```
+
+After disabling the exporter, no records will be exported to this exporter. Other exporters continues exporting.
```
Comment (lenaschoenburg) [inline docs/self-managed/zeebe-deployment/operations/management-api.md:97]:
```suggestion
After disabling the exporter, no records will be exported to this exporter. Other exporters continue exporting.
```

**Review by lenaschoenburg — APPROVED** [review]

**Comment by mesellings** [comment]
> > @deepthidevaki Which alpha/release is this for please?
> 
> If possible, include in the next 8.6 alpha (next week). Otherwise, in 8.6 minor.

Are you able to add the relevant label(s) for this please?

File: docs/self-managed/zeebe-deployment/operations/cluster-scaling.md (hunk 1 above)
Comment (deepthidevaki) [reply docs/self-managed/zeebe-deployment/operations/cluster-scaling.md:384]: There is no danger when force-removed brokers are suddenly reconnected.

Comment (deepthidevaki) [reply docs/self-managed/zeebe-deployment/operations/cluster-scaling.md]: I have improved the description for this request.

Comment (lenaschoenburg) [reply docs/self-managed/zeebe-deployment/operations/cluster-scaling.md]: Oh my bad, that's my half-knowledge about multi-region showing :sweat_smile:

**Review by conceptualshark — APPROVED** [review]
I've made a few edits for grammar and style, and rearranged a sentence or to for clarity - would you mind making sure I haven't lost any of the technical requirements in doing so? Otherwise, this is now good from me. 👍

**Comment by deepthidevaki** [comment]
Thanks @conceptualshark for the edits. Looks good to me. I will merge this PR.

**Pull request merged**